tee-worker/client-api/parachain-api/src/lookup.d.ts

```diff
@@ -0,0 +1,2 @@
+declare type Lookup76 = any;
```

Comment:
For xcmv5 support, polakdot.js-api devs have submitted a [PR](https://github.com/polkadot-js/api/pull/6142).

Reply:
FYI https://github.com/polkadot-js/api/releases/tag/v15.10.2
I think you could delete this declaration file and then regenerate the types to try again.